### PR TITLE
fixes for agent chart

### PR DESCRIPTION
**Pull request by wozniakjan**
mainly narrowing down rbac and additional configuration options

### CODE REVIEW

File: kedify-agent/templates/crd-install/crd-rbac.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ rules:
 - apiGroups:
   - ""
   resources:
+  # does this job really need to create jobs?
```
Comment (jkremser) [inline kedify-agent/templates/crd-install/crd-rbac.yaml:31]: good catch, it doesn't. Can you pls remove it in your pr?

```
- apiGroups:
  - ""
  resources:
  - jobs
  verbs:
  - create
  - delete
```

File: kedify-agent/templates/crd-install/crd-rbac.yaml [hunk 2]
```diff
@@ -41,6 +42,10 @@ rules:
   - delete
   - get
   - patch
+#  can this be restricted to only the CRDs agent needs?
+#  resourceNames:
+#  - kedifyconfigurations.install.kedify.io
+#  - scalingpolicies.keda.kedify.io
```
Comment (jkremser) [inline kedify-agent/templates/crd-install/crd-rbac.yaml:48]: create + resourceNames doesn't work

File: kedify-agent/values.yaml [hunk 3]
```diff
@@ -17,8 +17,8 @@ agent:
   createKedifyConfiguration: true
 
   image:
-    tag: "v0.0.53"
-    repository: ghcr.io/kedify/agent
+    tag: "1729243540"
+    repository: wozniakjan/agent
```
Comment (jkremser) [inline kedify-agent/values.yaml:21]: should this go in?

Comment (wozniakjan) [reply kedify-agent/values.yaml:21]: ah, right, shouldn't go in, just residue from testing https://github.com/josefkarasek/agent/pull/3

**Review by jkremser — APPROVED** [review]

**Review by jkremser — APPROVED** [review]

**Pull request merged**